luci-app-overthebox/luasrc/controller/admin/overthebox.lua

```diff
@@ -132,7 +132,7 @@ function interfaces_status()
 	end
 	-- Check overthebox service are running
 	mArray.overthebox["tun_service"] = false
-	if string.find(sys.exec("/usr/bin/pgrep -f ^glorytun"), "%d+") then
+	if string.find(sys.exec("/usr/bin/pgrep -f 'glorytun-(tcp|udp)'"), "%d+") then
```

Comment:
If i test for full path binarie (/usr/bin/glorytun-...) it will not work on the current "glorytun.sh" version (backward comp). So if toto-glorytun-udp-test.sh run a glorytun-test binary i don't care.
i just want to know if a glorytun is running